src/pages/filter/ui/filterAttributes.store.ts

```diff
@@ -104,6 +128,15 @@ export class FilterAttributesStore {
         selectedSubAttributes[variant[0]] = variant[1]
       }
     })
+
+    if (this.isAllMode) {
+      selectedSubAttributes['All'] = 1
```

Comment:
If you had 'currentMode' all you had to do is `selectedSubAttributes[this.currentMode] = 1`